src/routes/docs/get-started.svelte

```diff
@@ -48,31 +50,23 @@
 			<h3 class="text-2xl text-p5">Code</h3>
 			<CodeBlock code={gettingStartedExample} />
 		</section>
-		<h3 class="text-xl mb-3">Output</h3>
+		<h3 class="text-xl">Output</h3>
 
-		<label>
-			Width
-			<input type="range" bind:value={width} min="100" max="1000" step="0.01" />
-			{width}
-		</label>
-
-		<label>
-			Height
-			<input type="range" bind:value={height} min="100" max="1000" step="0.01" />
-			{height}
-		</label>
-
-		<P5 {sketch} />
+		<div class="border border-p5/40 rounded-md overflow-hidden mt-3">
+			<P5 {sketch} />
```

Comment:
[nit/idea for future] it'll be cool if we can get the sketch itself to resize to autofit the bounding box of its parent:

![image](https://user-images.githubusercontent.com/43280336/156687403-f3a775ed-3d55-4ee0-b25d-9a98cccaf0de.png)
![image](https://user-images.githubusercontent.com/43280336/156687453-c5aa8439-36ee-40b4-b9f6-8d4136ff18c6.png)

Reply:
How do you feel about adding a [prefix/suffix with the label](https://simeydotme.github.io/svelte-range-slider-pips/#prefixes-and-suffixes) for the above slider attributes? Maybe `x: `, `y: ` or emojis/unicode characters for the direction: ↕️ ↔️

Reply:
Tracking unique handleFormatter capability being supported in the range slider component https://github.com/simeydotme/svelte-range-slider-pips/issues/64

Reply:
Sorry for not answering earlier, I actually did all of that and then was looking at the CodeBlock component and started to play with highlight.js to implement code blocks in my personal projects 😅
Since I didn't take the time to come back and you already merged, I'll open a new PR to explain what I did and answer in the slider issue.